staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1365,6 +1381,12 @@ type VirtualMachineStatus struct {
 	// +nullable
 	// +optional
 	StartFailure *VirtualMachineStartFailure `json:"startFailure,omitempty" optional:"true"`
+
+	// MemoryDumpRequest tracks memory dump request phase and info of getting a memory
+	// dump to the given pvc
+	// +nullable
+	// +optional
+	MemoryDumpRequest *VirtualMachineMemoryDumpRequest `json:"memoryDumpRequest,omitempty" optional:"true"`
```

Comment:
What happens if a dump is requested while one is in progress?  Should this be a list like VolumeRequests?

Reply:
as long as the previous request hasn't completed the request is denied, there is not really a point to do another memory dump one after the other